lucida/commandcenter/controllers/dcm/IMMDCM.py

```diff
@@ -0,0 +1,19 @@
+import abc
+from Decision import*
+
+not_monument = 'cow'
+
+class IMMDCM(Decision):
+
```

Comment:
can you throw a doc string here describing a little what's going on. this is a template for DCM nodes so should be well documented.